src/test/cls_sem_set/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+add_executable(ceph_test_cls_sem_set
+  test_cls_sem_set.cc
+  )
+target_link_libraries(ceph_test_cls_sem_set
+  libneorados
+  ${BLKID_LIBRARIES}
+  ${CMAKE_DL_LIBS}
+  ${CRYPTO_LIBS}
+  ${EXTRALIBS}
+  neoradostest-support
+  ${UNITTEST_LIBS}
+  )
+install(TARGETS
+  ceph_test_cls_sem_set
```

Comment:
please make sure that all of these non-unit tests get added to teuthology somewhere. most of the cls ones are under qa/workunits/cls, while rgw-specific stuff can go under qa/workunits/rgw

the rgw/verify suite runs most of these from qa/suites/rgw/verify/tasks/cls.yaml

Reply:
Done.

Reply:
i don't see any changes under qa/ related to this part. what am i missing?